modules/openshift-architecture-common-terms.adoc

```diff
@@ -0,0 +1,186 @@
+
+OpenShift CLI (`oc`)::
+A command line tool to run {product-title} commands on the terminal.
+
+OpenShift Dedicated::
+A managed {op-system-base} {product-title} offering on Amazon Web Services (AWS) and Google Cloud Platform (GCP). OpenShift Dedicated focuses on building and scaling applications.
+
+{product-title} registry::
+A registry provided by {product-title} to manage images.
+
+Operator::
+The preferred method of packaging, deploying, and managing a Kubernetes application in an {product-title} cluster. An Operator takes human operational knowledge and encodes it into software that is packaged and shared with customers.
+
+OperatorHub::
+A platform that contains various {product-title} Operators to install.
+
+Operator Lifecycle Manager (OLM)::
+OLM helps you to install, update, and manage the lifecycle of Kubernetes native applications. OLM is an open source toolkit designed to manage Operators in an effective, automated, and scalable way.
+
+Over-the-air (OTA) updates::
+The {product-title} Update Service (OSUS) provides over-the-air updates to {product-title}, including {op-system-first}.
+
+pod::
+One or more containers with shared resources, such as volume and IP addresses, running in your {product-title} cluster.
+A pod is the smallest compute unit defined, deployed, and managed.
+
+private registry::
+{product-title} can use any server implementing the container image registry API as a source of the image which allows the developers to push and pull their private container images.
+
+public registry::
+{product-title} can use any server implementing the container image registry API as a source of the the image which allows the developers to push and pull their public container images.
+
+{op-system-base} {product-title} Cluster Manager::
+A managed service where you can install, modify, operate, and upgrade your {product-title} clusters.
+
+{op-system-base} Quay Container Registry::
+A Quay.io container registry that serves most of the container images and Operators to {product-title} clusters.
+
+replication controllers::
+An asset that indicates how many pod replicas are required to run at a time.
+
+Role-based access control (RBAC)::
+A key security control to ensure that cluster users and workloads have only access to resources required to execute their roles.
+
+route::
+Exposes a service to allow for network access to pods from users and applications outside the {product-title} instance.
+
+scaling::
+Increasing or decreasing the resource capacity.
+
+service::
+Exposes a running application on a set of pods.
+
+Source-to-Image (S2I) image::
+An image created based on the programming language of the application source code in {product-title} to deploy applications.
+
+storage::
+{product-title} supports many types of storage, both for on-premise and cloud providers. You can manage container storage for persistent and non-persistent data in an {product-title} cluster.
+
+telemetry::
```

Comment:
I'm pretty sure that Telemetry is capitalized (per how it's used here: https://docs.openshift.com/container-platform/4.11/support/remote_health_monitoring/about-remote-health-monitoring.html)

Reply:
addressed